samples/snippets/revoke_dataset_access_test.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from . import update_dataset_access
+from . import revoke_dataset_access
+from google.cloud import bigquery
+from functools import reduce
```

Comment:
1. These should be grouped into 3 parts: (a) standard library [built-ins], (b) third-party packages, (c) local packages https://www.python.org/dev/peps/pep-0008/#imports
2. Since there's no `__init__.py` in the `samples/snippets/` directory, relative imports don't work. We'll need to (a) add `__init__.py` and update the other tests to use relative imports [preferred end-goal, but probably should be a separate cleanup PR] or (b) use absolute imports, which work because our tests are run with `samples/snippets/` as the working directory.

https://source.cloud.google.com/results/invocations/8768d4f0-99fe-4520-8027-5be92e1db003/targets/cloud-devrel%2Fclient-libraries%2Fpython%2Fgoogleapis%2Fpython-bigquery%2Fsamples%2Fpython3.6%2Fpresubmit/log

```
update_dataset_access_test.py:15: in <module>
    from . import update_dataset_access
E   ImportError: attempted relative import with no known parent package
- generated xml file: /tmpfs/src/github/python-bigquery/samples/snippets/sponge_log.xml -
```

```suggestion
from functools import reduce

from google.cloud import bigquery

import update_dataset_access
import revoke_dataset_access
```